src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -402,15 +394,19 @@ void EnMa1_Update(Actor* thisx, PlayState* play) {
 
     Collider_UpdateCylinder(&this->actor, &this->collider);
     CollisionCheck_SetOC(play, &play->colChkCtx, &this->collider.base);
+
     SkelAnime_Update(&this->skelAnime);
     EnMa1_UpdateEyes(this);
+
     this->actionFunc(this, play);
+
     if (this->actionFunc != EnMa1_DoNothing) {
         Npc_UpdateTalking(play, &this->actor, &this->interactInfo.talkState, (f32)this->collider.dim.radius + 30.0f,
```

Comment:
Is the cast on `this->collider.dim.radius` required for matching? I don't think it should be needed so if not it would be nice to have those removed from several places

Reply:
It is not needed. This issue happens in quite a few different files, just early mips2c artifacts https://github.com/zeldaret/oot/pull/1595

No action needed, I made a PR to handle all of them (at least the ones related to `this->collider`)